src/main/java/uk/gov/hmcts/reform/em/stitching/config/security/SecurityConfiguration.java

```diff
@@ -44,19 +52,36 @@ public void configure(WebSecurity web) {
 
     @Override
     protected void configure(final HttpSecurity http) throws Exception {
-        // Don't erase user credentials as this is needed for the user profile
-        final ProviderManager authenticationManager = (ProviderManager) authenticationManager();
-        authenticationManager.setEraseCredentialsAfterAuthentication(false);
-        authCheckerFilter.setAuthenticationManager(authenticationManager());
-
-        http.antMatcher("/api/**")
-            .addFilter(authCheckerFilter)
-            .sessionManagement().sessionCreationPolicy(STATELESS).and()
-            .csrf().disable()
-            .formLogin().disable()
-            .logout().disable()
-            .authorizeRequests()
-            .anyRequest()
-            .authenticated();
+
+        http.csrf().disable()
+                .formLogin().disable()
+                .logout().disable()
+                .addFilterBefore(serviceAuthFilter, BearerTokenAuthenticationFilter.class)
```

Comment:
You will need JWTConverter to get roles, If you are using authorisation. Have a look at @smathangi 's PR. If you are using just authentication than this is ok. https://github.com/hmcts/ccd-data-store-api/pull/822/files